tests/main/interfaces-daemon-notify/task.yaml

```diff
@@ -19,15 +19,15 @@ execute: |
     snap connect test-snapd-daemon-notify:daemon-notify
 
     echo "Then after we restart the servive there is not denials"
-    denials_before="$(snap logs -n=100 test-snapd-daemon-notify.notify | grep -c 'Permission denied' || true)"
+    denials_before="$(snap logs -n=all test-snapd-daemon-notify.notify | grep -c 'Permission denied' || true)"
```

Comment:
Maybe it's a mistake to use `snap logs` after all. Since you added `get_journalctl_logs`, we should use that to ensure we're only browsing the logs from this test run. So the line should be changed to:
```
# shellcheck source=tests/lib/journalctl.sh
. "$TESTSLIB/journalctl.sh"
...
denials_before="$(get_journalctl_log -u snap.test-snapd-daemon-notify.notify.service | grep -c 'Permission denied' || true)"
```